style.css

```diff
@@ -427,7 +427,7 @@ li {
 }
 
 #contact {
-  background-color: #6070ff;
+  background-color: #525fbf;
   background-image: url(./images/contact\ form\ background\ shapes.png);
```

Comment:
- [ ] You are smart with your `background-image` here! 👍 The only challenge is using a file name with spaces in between makes it less accessible to users of screen readers as parsing the image name lacks clarity and this is also not a best practice. To name a file with more than one word, you may consider linking them with an underscore (`_`), an hyphen (`-`) or use camelCase, or PascalCasing. This will avoid potential bugs that may arise from using the files like not escaping the empty space, or forgetting the number of space, etc. As a good practice, kindly rename all your image files that do not follow the suggested methods.

👇 👇 👇 

![image](https://user-images.githubusercontent.com/99715640/214443184-c0beaa1b-4697-4ec9-9075-9a5fdb332d6b.png)